### DISCUSSION TITLE
Olympus and its (dubious) Game Theory

### DISCUSSION THREAD

**Discussion by rufuspollock** [rufuspollock/awesome-crypto-critique#226] [General]
[Olympus Game Theory - Olympus.pdf](https://github.com/life-itself/web3/files/13770841/Olympus.Game.Theory.-.Olympus.pdf)

**Comment by davidgasquez** [comment]
Heya! The PDF might be corrupt. This is what I'm getting.

![image](https://github.com/life-itself/web3/assets/1682202/768e892c-81c7-4e28-805b-e33406740556)

**Reply by rufuspollock** [reply]
@davidgasquez i only excerpted that at the time. I wanted to create a backup of their whole site for posterity. Any tips for how to do that quickly let me know (this were running off gitbook so i looked for the source but never found it publicly).

**Reply by davidgasquez** [reply]
I see! Perhaps "Internet Archive" has something to quickly scan and backup entire websites?
